config/24.7/ports.conf

```diff
@@ -96,6 +96,7 @@ net-mgmt/nrpe					arm
 net-mgmt/p5-FusionInventory-Agent
 net-mgmt/py-opn-cli
 net-mgmt/telegraf				arm
+net-mgmt/xymon-client
```

Comment:
for now I'd like to avoid "arm" build as well (that's the 32 bit arm, not aarch64)

Reply:
@fichtner done :heavy_check_mark: